### PR TITLE
Update logger to use remote logger

### PR DESCRIPTION
**Pull request by oferKatz07**
Removed logger HealthNodeHandler and added RemoteLogger from movai-common-enterprise repository

### CODE REVIEW

File: movai_core_shared/envvars.py [hunk 1]
```diff
@@ -46,6 +46,11 @@
 ROS1_USER_WS = os.getenv("ROS1_USER_WS")
 ROS1_WS = ROS1_MOVAI_WS
 ROS2_DISTRO = "dashing"
+# ZMQ environment variables
+MSG_HANDLER_LOCAL_CONN = os.getenv("MSG_HANDLER_LOCAL_CONN", "ipc:///run/movai/comm/msg_handler_local_comm.ipc")
+MOVAI_ZMQ_TIMEOUT_MS = os.getenv("MOVAI_ZMQ_TIMEOUT_MS", 1000)
```
Comment (Mograbi) [inline movai_core_shared/envvars.py:51]: isn't this considered a very low value for Timeout ?

File: movai_core_shared/logger.py [hunk 2]
```diff
@@ -7,21 +7,21 @@
...
 from movai_core_shared.envvars import (
     MOVAI_LOGFILE_VERBOSITY_LEVEL,
-    MOVAI_HEALTHNODE_VERBOSITY_LEVEL,
+    MOVAI_FLEET_LOGS_VERBOSITY_LEVEL,
     MOVAI_STDOUT_VERBOSITY_LEVEL,
     MOVAI_GENERAL_VERBOSITY_LEVEL,
     LOG_HTTP_HOST,
 )
+try:
+    from movai_core_enterprise import remote_logger
```
Comment (Mograbi) [inline movai_core_shared/logger.py:21]: remote_logger should exist in __init__.py for the enterprise in order to import it like this.

as I see in your branch https://github.com/MOV-AI/movai-core-enterprise/blob/Adding-message-handlers/movai_core_enterprise/__init__.py
it does not exist this way

**Review by Mograbi — CHANGES_REQUESTED** [review]

File: movai_core_shared/envvars.py (hunk 1 above)
Comment (oferKatz07) [reply movai_core_shared/envvars.py:51]: The value is in milliseconds and 1 second is a reasonable timeout for ZeroMQ

File: movai_core_shared/logger.py (hunk 2 above)
Comment (oferKatz07) [reply movai_core_shared/logger.py:21]: Fixed the import so that the __init__.py update is not necessary

**Review by engdorm — APPROVED** [review]

**Review by engdorm — APPROVED** [review]

**Pull request merged**